README.md

```diff
@@ -1,288 +1,176 @@
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# React-Capstone-Project <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+**React-Capstone-Project** This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that I create making use of React and Redux and follow the design by [Nelson Sakwa on Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).
 
-**[your_project__name]** is a...
+I select an API(https://disease.sh/v3/covid-19/countries) that provides Covid-19 data about the whole countries in the World and then build the webapp around it. The webapp has several pages:
+- Home page with a list of countrie that could be filtered by name.
 
 ## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+- JavaScript
+- React
+- Redux
 
 ### Tech Stack <a name="tech-stack"></a>
+- Git and Github
+- VS-code
+- JavaScript
+- HTM
+- CSS
+- React
+- Redux
+- Webpack
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+### Key Features <a name="key-features"></a>
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
+- **Select API**
+- **Build files structure for my React app.** 
+- **Prepare routes and navigation in my app**
+- **display a list of items and filter them**
+- **Create the tests for the application**
+- **Style my components to match the design provided**
+- **Deploy the project and test for final details**
+- **Create a good README and PR description.**
 
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
+
+**React front end framwork**
+- **Git-flow**
+- **Es6 module**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <details>
-<summary>Database</summary>
+  <summary>Client</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/React">React</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">JavaScript</a></li>
   </ul>
 </details>
-
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
+ 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [You can check the live demo here](https://covid-19-data-analysis.onrender.com)
```

Comment:
Your live demo links currently links to a 404 page. Please make sure your project can be accessed online
<img width="759" alt="image" src="https://github.com/Mikey1-nuel/React-Capstone-Project/assets/37116322/f08134f9-917c-4afc-96b0-d72ce25687df">